addons/purchase/tests/test_accrued_purchase_orders.py

```diff
@@ -45,6 +46,7 @@ def setUpClass(cls, chart_template_ref=None):
             'active_ids': cls.purchase_order.ids
         }).create({
             'account_id': cls.account_revenue.id,
+            'reversal_date': fields.Date.today() + relativedelta(days=1),
```

Comment:
Why would you need to provide the value in the tests? Shouldn't the compute function do it for you?

Reply:
BTW, in tests, don't use `.today()` but provide a real fixed date.
There are bugs that can appear when you run the test a certain day and those bugs can be hard to find.